web/src/main/java/no/nav/modiapersonoversikt/rest/persondata/PersondataService.kt

```diff
@@ -58,11 +58,17 @@ class PersondataServiceImpl(
         val tilganger = PersondataResult
             .runCatching(InformasjonElement.VEILEDER_ROLLER) { hentTilganger() }
             .getOrElse(PersondataService.Tilganger(kode6 = false, kode7 = false))
+        val kontaktinformasjonTredjepartsperson = PersondataResult.runCatching(InformasjonElement.DKIF) {
+            persondata
+                .findKontaktinformasjonTredjepartspersoner()
+                .map { dkif.hentDigitalKontaktinformasjon(it) }
+                .associateBy { it.personident }
+        }.getOrNull()
         val tredjepartsPerson = PersondataResult.runCatching(InformasjonElement.PDL_TREDJEPARTSPERSONER) {
             persondata
                 .findTredjepartsPersoner()
                 .let { pdl.hentTredjepartspersondata(it) }
-                .mapNotNull { tredjepartspersonMapper.lagTredjepartsperson(it.ident, it.person, tilganger) }
+                .mapNotNull { tredjepartspersonMapper.lagTredjepartsperson(it.ident, it.person, tilganger, kontaktinformasjonTredjepartsperson) }
```

Comment:
`kontaktinformasjonTredjepartsperson[it.ident]` er vel all informasjonen vi trenger å sende videre